tmc-prototype/DishLeafNode/DishLeafNode/DishLeafNode.py

```diff
@@ -347,156 +398,211 @@ def write_activityMessage(self, value):
+            self._dish_proxy.command_inout_asynch(CONST.CMD_DISH_SLEW, "0", self.commandCallback)
+        except Exception as except_occurred:
+            print CONST.ERR_EXE_CONFIGURE_CMD, except_occurred
+            self._read_activity_message = CONST.ERR_EXE_CONFIGURE_CMD +  str(except_occurred)
+            self.devlogmsg(CONST.ERR_EXE_CONFIGURE_CMD, 2)
 
         # PROTECTED REGION END #    //  DishLeafNode.Configure
 
     def is_Configure_allowed(self):
         # PROTECTED REGION ID(DishLeafNode.is_Configure_allowed) ENABLED START #
-        return self.get_state() not in [DevState.INIT,DevState.ALARM,DevState.DISABLE,DevState.OFF,DevState.STANDBY]
+        """ This method is an internal construct of TANGO """
+        return self.get_state() not in [DevState.INIT, DevState.ALARM, DevState.DISABLE,
+                                        DevState.OFF, DevState.STANDBY]
         # PROTECTED REGION END #    //  DishLeafNode.is_Configure_allowed
 
     @command(
-    dtype_in='str', 
-    doc_in="The timestamp indicates the time, in UTC, at which command execution should start.", 
+        dtype_in='str',
+        doc_in="The timestamp indicates the time, in UTC, at which command execution should start.",
     )
     @DebugIt()
     def StartCapture(self, argin):
         # PROTECTED REGION ID(DishLeafNode.StartCapture) ENABLED START #
+        """
+        This method sends the command to Start capture on the configured band.
+        Command only valid in SPFRx Data_Capture mode.
+        :param argin: timestamp
+        :return:
+        """
         try:
-            self._dish_proxy.command_inout_asynch("StartCapture", argin, self.commandCallback)
-        except Exception as e:
-            print "Exception in StartCapture command:", e
-            self._read_activity_message = "Exception in StartCapture command:" + str(e)
-            self.devlogmsg("Exception occurred in StartCapture command.", 2)
+            self._dish_proxy.command_inout_asynch(CONST.CMD_START_CAPTURE,
+                                                  argin, self.commandCallback)
+        except Exception as except_occurred:
+            print CONST.ERR_EXE_START_CAPTURE_CMD, except_occurred
+            self._read_activity_message = CONST.ERR_EXE_START_CAPTURE_CMD\
+                                          + str(except_occurred)
+            self.devlogmsg(CONST.ERR_EXE_START_CAPTURE_CMD, 2)
         # PROTECTED REGION END #    //  DishLeafNode.StartCapture
 
     @command(
-    dtype_in='str', 
-    doc_in="The timestamp indicates the time, in UTC, at which command execution should start.", 
+        dtype_in='str',
+        doc_in="The timestamp indicates the time, in UTC, at which command execution should start.",
     )
     @DebugIt()
     def StopCapture(self, argin):
         # PROTECTED REGION ID(DishLeafNode.StopCapture) ENABLED START #
+        """
+        This method sends the command to Stop capture on the configured band.
```

Comment:
...method sends the command.. Inappropriate description.

Reply:
Updated code documentation for CentralNode, SubarrayNode, DishLeafNode and DishMaster. Commit ID is ef9a34b.